src/gpodder/gtkui/desktop/preferences.py

```diff
@@ -805,10 +808,20 @@ def on_btn_playlist_folder_clicked(self, widget):
         fs.destroy()
 
     def on_checkbutton_use_proxy_toggled(self, widget):
+        widgets = (self.grid_network_proxy_details,
+                   self.vbox_network_proxy_username_password)
         if widget.get_active():  # Enable the proxy input details
-            self.grid_network_proxy_details.set_sensitive(True)
+            for w in widgets:
+                w.set_sensitive(True)
         else:  # Disable
-            self.grid_network_proxy_details.set_sensitive(False)
+            for w in widgets:
+                w.set_sensitive(False)
```

Comment:
This could be done with a single for loop and `w.set_sensitive(widget.get_active())`.